web/src/Web.App/Views/SchoolComparators/Index.cshtml

```diff
@@ -53,12 +54,48 @@
                     <li>pupils eligible for free school meals (FSM)</li>
                     <li>pupils with special educational needs (SEN)</li>
                 </ul>
-                <h3 class="govuk-heading-s">Schools we've chosen</h3>
+                <table class="govuk-table">
```

Comment:
This table looks very familiar! I might refactor into a partial as part of my current work and include an optional extra column where I need to support the 'removal' from a user defined set